scripts/variantstore/wdl/LoadBigQueryData.wdl

```diff
@@ -0,0 +1,187 @@
+    Int? preemptible_tries
+    String docker
+    String? for_testing_only
+  }
+
+  call CreateTables {
+  	input:
+      project_id = project_id,
+      dataset_name = dataset_name,
+      storage_location = storage_location,
+      datatype = datatype,
+      max_table_id = max_table_id,
+      schema = schema,
+      numbered = numbered,
+      partitioned = partitioned,
+      load = load,
+      uuid = uuid,
+      done = done,
+      preemptible_tries = preemptible_tries,
+      docker = docker,
+      for_testing_only = for_testing_only
+  }
+
+  scatter (table_dir_files_str in CreateTables.table_dir_files_list) {
+    call LoadTable {
+      input:
+        table_dir_files_str = table_dir_files_str,
+        project_id = project_id,
+        schema = schema,
+        load = load,
+        preemptible_tries = preemptible_tries,
+        docker = docker
+    }
+  }
+
+  output {
+    Array[String] table_dir_files_list = CreateTables.table_dir_files_list
+  }
+}
+
+task LoadTable {
+  meta {
+    volatile: true
+  }
+
+  input {
+    String table_dir_files_str
+    String project_id
+    File schema
+    String load
+
+    Int? preemptible_tries
+    String docker
+  }
+
+  command <<<
+    TABLE=$(echo ~{table_dir_files_str} | cut -d, -f1)
+    DIR=$(echo ~{table_dir_files_str} | cut -d, -f2)
+    FILES=$(echo ~{table_dir_files_str} | cut -d, -f3)
+
+    #load should be false if using Google Storage Transfer so that the tables will be created by this script, but no data will be uploaded.
+    if [ ~{load} = true ]; then
+      bq load --location=US --project_id=~{project_id} --skip_leading_rows=1 --source_format=CSV -F "\t" $TABLE $DIR$FILES ~{schema}
+      echo "ingested ${FILES} file from $DIR into table $TABLE"
+      gsutil mv $DIR$FILES ${DIR}done/
+    else
+      echo "${FILES} will be ingested from $DIR by Google Storage Transfer"
+    fi
+  >>>
+
+  runtime {
+    docker: docker
+    memory: "3 GB"
+    disks: "local-disk 10 HDD"
+    preemptible: select_first([preemptible_tries, 5])
+    cpu: 1
+  }
+}
+
+# Creates all the tables necessary for the LoadData operation
+# As an optimization, I also generate a (table, dir, files) csv file which contains
+# most of inputs necessary for the following LoadTable task.
+task CreateTables {
+	meta {
+    	volatile: true
+  	}
+
+	input {
+      String project_id
+      String dataset_name
+      String storage_location
+      String datatype
+      Int max_table_id
+      File schema
+      String numbered
+      String partitioned
+      String load
+      String uuid
+
+      #input from previous task needed to delay task from running until the other is complete
+      Array[String] done
+
+      # runtime
+      Int? preemptible_tries
+      String docker
+
+      String? for_testing_only
+    }
+
+  command <<<
+    set -x
+    set -e
+    ~{for_testing_only}
+
+    DIR="~{storage_location}/~{datatype}_tsvs/"
+
+    for TABLE_ID in $(seq 1 ~{max_table_id}); do
```

Comment:
I think this is good for now, but I think we should change getMaxTableId to getTableIdsForSamples that returns just the list of tables that we need for the current samples and then we loop through those. I'll add a feature request for this.